pages/account/index.vue

```diff
@@ -134,17 +133,17 @@
         likedItems: state => state.set.likedItems
       })
     },
-
     watch: {
       likesId: 'fetchLikes'
     },
-
+    mounted() {
+      this.$store.dispatch('set/fetchCreations');
+    },
     methods: {
       fetchLikes() {
-        this.$store.dispatch('set/fetchLikes');
+        return this.$store.dispatch('set/fetchLikes');
```

Comment:
I don't think this needs to ever return anything. The return value isn't ever assigned, all the uses are watching the store value directly.